hw/ip/rom_ctrl/util/scramble_image.py

```diff
@@ -265,7 +269,10 @@ def from_hjson_path(path: str, mode: str, hash_file: IO[str]) -> 'Scrambler':
         key = Scrambler._get_param_value(params,
                                          mc_params.scr_key,
                                          mc_params.scr_key_width)
-        return Scrambler(nonce, key, base, size_words, hash_file)
+        nonce_width = mc_params.nonce_width
+        return Scrambler(nonce, nonce_width,
+                         key, mc_params.scr_key_width,
```

Comment:
I think it's probably best to do the same thing for both widths. Maybe make the second argument `mc_params.nonce_width` ?

Reply:
Yes, line 272 is useless. Fixed as well, thanks.